include/deal.II/fe/fe_values.h

```diff
@@ -2539,13 +2539,12 @@ class FEValuesBase :
    */
   DeclException1 (ExcAccessToUninitializedField,
                   char *,
-                  << ("You are requesting information from an FEValues/FEFaceValues/FESubfaceValues "
-                      "object for which this kind of information has not been computed. What "
-                      "information these objects compute is determined by the update_* flags you "
-                      "pass to the constructor. Here, the operation you are attempting requires "
-                      "the <")
-                  << arg1
-                  << "> flag to be set, but it was apparently not specified upon construction.");
+                  << "You are requesting information from an FEValues/FEFaceValues/FESubfaceValues "
+                  << "object for which this kind of information has not been computed. What "
```

Comment:
yes, but because I am not parsing the c++ code in the javascript code, the parentheses will be part of the message (see https://www.dealii.org/developer/doxygen/deal.II/group__Exceptions.html#ga10ed08551ffd294db194fa94d873cf83)